evalml/data_checks/class_imbalance_data_check.py

```diff
@@ -22,9 +24,11 @@ class ClassImbalanceDataCheck(DataCheck):
         min_samples (int): The minimum number of samples per accepted class. If the minority class is both below the threshold and min_samples,
             then we consider this severely imbalanced. Must be greater than 0. Defaults to 100.
         num_cv_folds (int): The number of cross-validation folds. Must be positive. Choose 0 to ignore this warning. Defaults to 3.
+        test_size (None, float, int): Percentage of test set size. Used to calculate class imbalance prior to splitting the
+            data into training and validation/test sets.
```

Comment:
Hyper nit - won't you need a `Raises:` block for the new ValueError?  I guess not given the existing ValueError isn't complained about.

Reply:
+1 to ValueError, lame that it's not caught by our linter because we move our docstring from __init__ to our class docstring (IIRC for docs) so it doesn't automatically pick it up 😢

Reply:
My guess is that darglint doesn't check for raises blocks in constructors.  @angela97lin do you know?

Reply:
Yeah @freddyaboulton, I don't think it does a good job for class docstrings in general (I think I recently came across a case where we added a new parameter but it didn't complain about the missing docstring).

If we had this docstring in the constructor (__init__) it might actually pick it up, but IIRC, then our docs won't look like this: https://evalml.alteryx.com/en/stable/autoapi/evalml/data_checks/class_imbalance_data_check/index.html#evalml.data_checks.class_imbalance_data_check.ClassImbalanceDataCheck

We'd have to add back __init__ and click on the __init__ method link specifically to see the parameters, rather than just at the class page.